app/src/main/java/io/gnosis/safe/ui/settings/owner/ledger/LedgerDeviceListFragment.kt

```diff
@@ -184,8 +183,8 @@ class LedgerDeviceListFragment : BaseViewBindingFragment<FragmentLedgerDeviceLis
         super.onRequestPermissionsResult(requestCode, permissions, grantResults)
         viewModel.handlePermissionResult(
             this,
-            ::handleMissingLocationPermission,
-            ::requestMissingLocationPermission,
+            ::handleMissingPermission,
```

Comment:
what is the difference btw :handleMissingPermission and ::requestMissingBLEPermission? Do we need this separation?